scripts/install-pedatious.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# exit on any error
+
+#STILL UNDER CONSTRUCTION
+set -Ee
+
+PEDATIOUS_DIR=pedatious
+CORES=$(nproc)
+PEDATIOUS_VERSION=0.1
+
+# for ubuntu precise in travis, that does not provide pkg-config:
+pkg-config --exists icu-i18n || export CFLAGS="$CFLAGS -I/usr/include/x86_64-linux-gnu"
+pkg-config --exists icu-i18n || export LDFLAGS="$LDFLAGS -licui18n -licuuc -licudata"
+
+
+#install pre-reqs for building
+sudo apt-get install python3 ninja-build build-essential \
+    && pip3 install --user meson
+
+# download and install pedatious
+if [ ! -d ${PEDATIOUS_DIR} ]; then
+    git clone --branch ${PEDATIOUS_VERSION} https://github.com/MycroftAI/pedatious.git
+    cd ${PEDATIOUS_DIR}
+    make -j$CORES
```

Comment:
Here, and below, instead of using make, it uses:
```
meson build
cd build
ninja clean # Optional; same as make clean
ninja
```

As a result, when using `ninja`, the correct amount of cores are used automatically.